{
  "modified": "2024-08-11T16:48:00Z",
  "published": "2024-08-11T16:48:00Z",
  "schema_version": "1.5.0",
  "id": "",
  "summary": "Malicious code in artifact-lab-3-package-392c6acd (PyPI)",
  "details": "The OpenSSF Package Analysis project identified 'artifact-lab-3-package-392c6acd' @ 0.1.2 (pypi) as malicious.\n\nIt is considered malicious because:\n\n- The package communicates with a domain associated with malicious activity.\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "artifact-lab-3-package-392c6acd"
      },
      "versions": [
        "0.1.2"
      ]
    }
  ],
  "credits": [
    {
      "name": "OpenSSF: Package Analysis",
      "type": "FINDER",
      "contact": [
        "https://github.com/ossf/package-analysis",
        "https://openssf.slack.com/channels/package_analysis"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "ossf-package-analysis",
        "sha256": "4919fee24d801edd7b2c72355220531b85d866e2b1d3aa1356ecf96d502a8b3e",
        "import_time": "2024-09-18T02:42:07.426334837Z",
        "modified_time": "2024-08-11T16:48:00Z",
        "versions": [
          "0.1.2"
        ]
      }
    ]
  }
}
